src/test/resources/com/uber/jenkins/phabricator/conduit/validFetchDiffResponse.json

```diff
@@ -5,7 +5,18 @@
       "sourceControlBaseRevision": "4b825dc642cb6eb9a060e54bf8d69288fbee4904",
       "hello": "world",
       "authorName": "aiden",
-      "authorEmail": "sc@ndella.com"
+      "authorEmail": "sc@ndella.com",
+      "changes": [
+        {
+          "currentPath": "github.com/uber/go-torch/visualization/visualization.go"
```

Comment:
So, I think this might actually break things for Go, since the "phabricator" path will be everything **after** the URI portion (e.g. `visualization/visualization.go`)